Algorithm-Exact-Change.md

```diff
@@ -98,8 +110,22 @@ checkCashRegister(19.50, 20.00, [["PENNY", 1.01], ["NICKEL", 2.05], ["DIME", 3.1
 
 - Read comments in code.
```

Comment:
No, the comments in code are optional, preferred but the code explanation after the code is a must. We didn't enforce that or anything before, but that is the purpose of these changes, make it standard and follow a set of rules to avoid having a bunch of similar solutions with no real explanation.